docs/howto/catalog_exports.md

```diff
@@ -0,0 +1,149 @@
+---
+title: Data Catalogs Export
+description: This section explains how lakeFS can integrate with external Data Catalogs via metastore update operations. 
+parent: How-To
+---
+
+# Data Catalogs Export
+
+{% include toc_2-3.html %}
+
+## About Data Catalogs Export
+
+Data Catalog Export is all about integrating external Data Warehouses (i.e AWS Athena) with lakeFS.
+
+Data Catalogs such as Hive, Glue and other store metadata for services (such as Spark, Trino and Athena). They contain metadata such as the location of the table, information about columns, partitions and much more.
+Given the native integration between Spark and SQL, it’s most common that you’ll interact with tables in Spark environment.
+
+While different systems support reading directly from lakefs (i.e Gateway endpoint `s3a://`) many systems only support direct access to blockstore (i.e S3 bucket). 
+
+With Data Catalog Exports, one can leverage the versioning capabilities of lakeFS in external data warehouses and manage tables with branches and commits. 
+
+Once hooks are set up, querying lakeFS data from e.g. Athena, Trino and other catalog-dependant tools looks like this:
```

Comment:
```suggestion
At the end of this guide, you will be able to query lakeFS data from Athena, Trino and other catalog-dependent tools:
```
(because hooks were not mentioned yet).